sdk/src/main/java/com/hedera/hashgraph/sdk/TransactionResponse.java

```diff
@@ -62,6 +62,13 @@ public TransactionReceipt getReceipt(Client client) throws TimeoutException, Pre
         return receipt;
     }
 
+    public TransactionReceipt forceGetReceipt(Client client) throws TimeoutException, PrecheckStatusException {
+        return new TransactionReceiptQuery()
+            .setTransactionId(transactionId)
+            .setNodeAccountIds(Collections.singletonList(nodeId))
+            .execute(client);
+    }
+
```

Comment:
You know what?  I am dumb.  I'm pretty sure the async versions aren't necessary.

Reply:
Removed